tests/core/pyspec/eth2spec/gen_helpers/gen_base/gen_runner.py

```diff
@@ -127,12 +133,25 @@ def run_generator(generator_name, test_providers: Iterable[TestProvider]):
                 / Path(test_case.suite_name) / Path(test_case.case_name)
             )
             if case_dir.exists():
-                if not args.force:
+                # Check if the test vector was generated successfully.
+                meta_file = case_dir / 'meta.yaml'
+                # Check if meta file exists
+                meta_dict = {}
+                if meta_file.exists():
+                    with meta_file.open('r') as f:
+                        meta_dict = safe_load(f.read())
+
+                if 'release_version' in meta_dict and not args.force:
```

Comment:
wouldn't it make more sense to check the version value, instead of the presence of the key?